middlewares/hacker.middleware.js

```diff
@@ -492,6 +492,18 @@ async function updateHacker(req, res, next) {
     }
 }
 
+/**
+ * Sets Hacker Status to Accepted and runs it through updateHacker's functionality.
+ * @param {{params:{id: string}, body: *}} req 
+ * @param {*} res 
+ * @param {*} next 
+ */
+async function acceptHacker(req, res, next) {
+    req.body.status = Constants.General.HACKER_STATUS_ACCEPTED;
+    updateHacker(req, res, next);
```

Comment:
Nit: Since this is a middleware possibly, just put it under the route and remove this line?

Reply:
To add onto Harsh's comment:
```javascript
function parseAccept(req, res, next) {
    req.body.status = Constants.General.HACKER_STATUS_ACCEPTED;
    next();
}
```
and then in routes/hacker.js:
```javascript
...
Middleware.Hacker.parseAccept,
Middleware.Hacker.updateHacker,
...
```